src/main/java/net/geforcemods/securitycraft/datagen/BlockTagGenerator.java

```diff
@@ -257,10 +297,16 @@ protected void registerTags()
 				SCContent.REINFORCED_GREEN_STAINED_GLASS.get(),
 				SCContent.REINFORCED_RED_STAINED_GLASS.get(),
 				SCContent.REINFORCED_BLACK_STAINED_GLASS.get());
+		getOrCreateBuilder(BlockTags.INFINIBURN_OVERWORLD).add(SCContent.REINFORCED_NETHERRACK.get());
+		getOrCreateBuilder(BlockTags.NYLIUM).addTag(SCTags.Blocks.REINFORCED_NYLIUM);

```

Comment:
Reinforced Nylium isn't Nylium. We don't add Reinforced Cobblestone to Forge's cobblestone tag either.

Reply:
Thing is, I need to add reinforced nylium to this tag because the nether plants (nether roots, nether sprouts) that should appear when bonemealing the nylium blocks only sustain on blocks if they're in that tag.